python/lsst/ts/aos/utils/m1m3/M1M3LUT.py

```diff
@@ -39,14 +40,15 @@ class ForceType(enum.IntEnum):
 
 
 async def update_lut_force_balance(
-    force_type,
-    start_time,
-    end_time,
-    axis,
-    lut_path,
-    polynomial_degree,
-    resample_rate,
-    static_offset=1,
+    force_type: ForceType,
+    start_time: Time,
+    end_time: Time,
+    axis: str,
```

Comment:
Fix to axes

Reply:
No, as this is only one axis - the loop over axis is in the run method. Yes, we can do better by storing elevation data and not quering and resampling it again..